client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,127 @@
+<div>
+
+    <div class="widget-presentation">
+        <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title">
+        </h2>
+
+        <tuc-toast-message></tuc-toast-message>
+
+        <div ng-if="$ctrl.isRMAOpened">
+            <div data-translate="xdsl_access_comfort_exchange_already_opened_rma"></div>
+            <br/>
+            <div ng-repeat="rma in $ctrl.rmas track by $index">
+                <div class="form-group row">
+                    <label class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="xdsl_access_comfort_exchange_rma_status"
+                        for="rmaStatus"></label>
```

Comment:
`rmaStatus` refers to a `span` but it should refer to a labelable item, for example `button`, `input`, `select` or `textarea`